tests/Functional/Schema/MySQL/ComparatorTest.php

```diff
@@ -124,8 +124,8 @@ public function testExplicitDefaultCollation(): void
     public function testChangeColumnCharsetAndCollation(): void
     {
         [$table, $column] = $this->createCollationTable();
-        $column->setPlatformOption('charset', 'utf8');
-        $column->setPlatformOption('collation', 'utf8_bin');
+        $column->setPlatformOption('charset', 'latin1');
+        $column->setPlatformOption('collation', 'latin1_bin');
```

Comment:
From the [documentation](https://mariadb.com/kb/en/unicode/):

> [MariaDB 10.6](https://mariadb.com/kb/en/what-is-mariadb-106/), `utf8` is an alias for `utf8mb3`.

In order to test schema introspection, we should use a charset that isn't an alias so that it's introspected as is.